gssapi/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "gss-api"
+version = "0.0.0"
+description = """
+Pure Rust implementation for Generic Security Service Application Program Interface
+"""
+authors = ["RustCrypto Developers"]
+license = "Apache-2.0 OR MIT"
+repository = "https://github.com/RustCrypto/formats/tree/master/gssapi"
+categories = ["cryptography", "data-structures", "encoding", "no-std", "parser-implementations"]
+keywords = ["crypto", "key", "gssapi", "rfc1509", "rfc2478", "rfc4178", "ms-spng"]
```

Comment:
crates.io allows a maximum of 5 keywords

Reply:
I'm fine with `gss` and `gssapi` in there. Rest is optional/not necessary in my opinion. Any keywords you would still like to have? Maybe leave `crypto` in?

Reply:
`kerberos` would probably make sense.

Re: categories, perhaps `authentication` would make sense. You could possibly get rid of `data-structures`, as I believe that's more for `std::collections`-style data structures than protocol messages.